core/standalone/build.gradle

```diff
@@ -37,30 +38,41 @@ scoverage {
     scoverageScalaVersion.set("${gradle.scala.scoverageScalaVersion}")
 }
 
+task downloadSwagger(type: Download) {
+    src "https://github.com/swagger-api/swagger-ui/archive/v${gradle.swagger.version}.tar.gz"
```

Comment:
Could you check that the download is not happening if the file is present?

Reply:
Yes, that appear to work properly.
Running a `clean build -x test` followed by a `build` doesn't download anything.